model/registry/db.yaml

```diff
@@ -529,3 +533,67 @@ groups:
           reference the [elasticsearch schema](https://raw.githubusercontent.com/elastic/elasticsearch-specification/main/output/schema/schema.json)
           in order to map the path part values to their names.
         examples: ['db.elasticsearch.path_parts.index=test-index', 'db.elasticsearch.path_parts.doc_id=123']
+  - id: registry.db.vector
+    prefix: db.vector
+    type: attribute_group
+    brief: >
+      This group defines attributes for vector databases.
+    attributes:
+      - id: similarity
+        type:
+          allow_custom_values: true
+          members:
+            - id: cosine
+              value: 'cosine'
+              brief: >
+                The cosine metric.
+              stability: experimental
+            - id: dot
+              value: 'dot'
+              brief: >
+                The dot product metric.
+              stability: experimental
+            - id: euclidean
+              value: 'euclidean'
+              brief: >
+                The euclidean distance metric.
+              stability: experimental    
+            - id: manhattan
+              value: 'manhattan'
+              brief: >
+                The Manhattan distance metric.
+              stability: experimental
+        stability: experimental
+        brief: >
+          The metric used in similarity search.
+        examples: 'cosine'
+      - id: id
+        type: string
+        stability: experimental
+        brief: >
+          The ID of vector.
+        examples: '5c56c793-69f3-4fbf-87e6-c4bf54c28c26'
+      - id: name
+        type: string
+        stability: experimental
+        brief: >
+          The name field as of the vector (e.g. a field name).
+        examples: 'vector'
+      - id: embeddings
+        type: double[]
+        stability: experimental
+        brief: >
+          The values of the vector, the array of numbers.
+        examples: [0.9, 0.1, 0.1]
+      - id: dimension
+        type: int
+        stability: experimental
+        brief: >
+          The dimension of the vector.
```

Comment:
is it the number of dimensions? let's call it something like `db.vector.dimension_count` and also update the brief

Reply:
@ezimuel you marked this as resolved, but looks like you missed this change, it still shows `dimension` instead of `dimention_count`

Reply:
Fixed in https://github.com/open-telemetry/semantic-conventions/pull/1231/commits/a3330ffea04ad5d2ce8d9a4bb07a1ce259369339